.github/workflows/pr-check.yml

```diff
@@ -15,28 +15,23 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Validate Gradle wrapper
         uses: gradle/wrapper-validation-action@v1
 
       - name: Set up Java 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           java-version: 11
           distribution: 'zulu'
 
       - name: Set up Gradle cache
-        uses: actions/cache@v1
-        with:
-          path: ~/.gradle/caches
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle.kts') }}
-          restore-keys: |
-            ${{ runner.os }}-gradle-
```

Comment:
`gradle/gradle-build-action` just caches the `.gradle` folder, `.m2` folder is handled by [actions/cache](https://github.com/Goooler/graphql-kotlin/blob/b3ae3449dbe968e576406bab5632244dbf92d600/.github/workflows/pr-check.yml#L34-L39).